libhb/qsv_common.c

```diff
@@ -2291,8 +2312,6 @@ void hb_qsv_force_workarounds()
 #undef FORCE_WORKAROUNDS
 }
 
-AVBufferRef *enc_hw_frames_ctx = NULL;
-extern EncQSVFramesContext hb_enc_qsv_frames_ctx;
 AVBufferRef *hb_hw_device_ctx = NULL;
 char *qsv_device = NULL;
```

Comment:
Not part of this PR, but this appears to never be set to anything except NULL and never used because it is NULL.

I suggest adding a TODO for future cleanup